### PR TITLE
feature: SUSE AI sanity test suite

### PR DESCRIPTION
**Pull request by rqueiroz-suse**
Adding wrapper script to call SUSE AI sanity test suite

- Related ticket: https://jira.suse.com/browse/SUSEAI-8
- Needles: N/A
- Verification run: https://openqa.suse.de/tests/overview?distri=sle-micro&version=6.0&build=0153&groupid=592

### CODE REVIEW

**Review by asmorodskyi — APPROVED** [review]
LGTM

**Comment by m-dati** [comment]
Hi, it LGTM, but first, please your code seems needing perl tidy, then be sure to do git rebase master too: kindly, update your commit.
This apart, LGTM

**Review by m-dati — COMMENTED** [review]
Please check my comments

**Review by ricardobranco777 — COMMENTED** [review]
Please add a header including copyright, module description, etc.

**Comment by m-dati** [comment]
You sould run a perl tidy of changed modules, before git-push :slightly_smiling_face: 
Some `trailing spaces` claimed by workflow checks

**Review by ricardobranco777 — APPROVED** [review]

File: tests/publiccloud/aistack_sanity_run.pm [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    my $admin_password = get_var('OPENWEBUI_ADMIN_PWD');
+
+    assert_script_run("curl -O " . $sanity_tests_url);
+    assert_script_run("mkdir " . $test_folder);
+    assert_script_run("tar -xzvf open-webui-sanity-tests.tar.gz -C " . $test_folder);
+    assert_script_run("python3.11 -m venv " . $test_folder . "/venv"); 
+    assert_script_run("source " . $test_folder . "/venv/bin/activate");
+    assert_script_run("pip3 install -r ./" . $test_folder . "/requirements.txt");
+    assert_script_run("cp " . $test_folder . "/env.example " . $test_folder . "/.env");
+    assert_script_run("pytest --URL='https://$host_name' --OPENWEBUI-ADMIN-EMAIL=$admin_email --OPENWEBUI-ADMIN-PWD=$admin_password $test_folder/tests/");
```
[the remark this answers is not in the document]
Comment (rqueiroz-suse) [reply tests/publiccloud/aistack_sanity_run.pm]: Changed the way that the tests uses environment variables. Now, the os.getenv will get the predefined variables.

File: tests/publiccloud/aistack_sanity_run.pm [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+sub run {
+    my ($self, $args) = @_;
+
+    my $instance = $self->{my_instance};
+    my $provider = $self->{provider};
+
+    my $sanity_tests_url = data_url("aistack/open-webui-sanity-tests.tar.gz");
+    my $test_folder = "open-webui-sanity-tests";
```
[the remark this answers is not in the document]
Comment (rqueiroz-suse) [reply tests/publiccloud/aistack_sanity_run.pm]: Applied the approach where the wrapper will call the environment variable that has the archive name.

**Review by m-dati — APPROVED** [review]

**Comment by asmorodskyi** [comment]
because I just merged https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20724 you need to rebase now

**Comment by asmorodskyi** [comment]
other than that it is ready to be merged

**Pull request closed without merging**